packages/core/stories/hooks/useMedia.story.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { useEffect, useState } from 'react';
 import { Story } from '@storybook/react/types-6-0';
-import { QX_DEVICE, QxDevice, useMedia, useTheme } from '@core';
 import styled from '@emotion/styled';
-import { deviceToBreakpoint } from '../../styles/engine/theme/hooks/useMedia/helpers';
+import { QX_DEVICE, QxDevice, useMedia, useTheme } from '@core';
+import { deviceToBreakpoint } from '@core/styles/engine/theme/hooks/useMedia/helpers';
 
 export default {
-    title: 'hooks/useMedia',
+    title: 'core/hooks/useMedia',
```

Comment:
Может стоит вынести эти пути в константу/функцию, тогда будет проще эти пути менять без необходимости идти в каждый стори-файл:
```suggestion
    title: `${STORY_PATHS.hooks}/useMedia`,
```
Над названием, конечно, можно подумать

Reply:
Да, отлично, добавлю